exercises/react-elements/src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const helloReact = React.createElement(
+  'h1',
+  [null],
+  ['Hello, React!']
```

Comment:
You can leave out the square brackets. In the React documentation they mean "optional"